src/list-sorting/index.js

```diff
@@ -1,3 +1,16 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+  // Iterating backwards through haystack
+  for (let item = haystack.length - 1; item >= 0; item--) {
+    if (Array.isArray(haystack[item])) {
+      // finding and returning column and row values if there is a match
+      if (haystack[item].lastIndexOf(needle) > -1) {
+        return [item, haystack[item].lastIndexOf(needle)];
```

Comment:
Nice hack 😁 

`item` should've been `row` though, that's a better name.

Reply:
Thank ya.

You're right.
Heat of the moment coding :satisfied:
Will do better next time.